integration/helpers/utils.ts

```diff
@@ -22,10 +21,12 @@ export function setup(): Application {
 
 export async function tearDown(app: Application) {
   const pid = app.mainProcess.pid
+  let ppid: any = pid
+  ppid = await ppid()
```

Comment:
This looks like very bad form. Assigning and changing type.
```suggestion
  const pid = await app.mainProcess.pid()
```

Reply:
hmm... now after merging remembered that this might not work. Typings says thet app.mainProcess.pid is number, so we can't explicitly call it as function.

Reply:
If it is a number then trying to call it would result in a runtime error.

Reply:
No it's not a number despite typings say so.